### ISSUE TITLE
Consider adding compatibility table/badge

### ISSUE THREAD

**Issue by djeedai** [JonahPlusPlus/bevy_atmosphere#34]
It's a bit unclear which version of `bevy_atmosphere` is compatible with which version of Bevy. I would suggest following Bevy's own [Plugin Guidelines](https://github.com/bevyengine/bevy/blob/main/docs/plugins_guidelines.md#main-branch-tracking) and adding a badge and/or a compatibility table to help users pick the right version of `bevy_atmosphere` more easily 😊

**Comment by djeedai** [comment]
Case in point, I just got caught by it looking at the `Cargo.toml` on GitHub and seeing "great, it's on 0.9", but the published crate is on 0.8.1 instead.

**Comment by JonahPlusPlus** [comment]
Ah, I get your point. I'll remove the badge temporarily.

**Comment by JonahPlusPlus** [comment]
Yeah, I forgot I was working on main instead of a 0.5 branch. Sorry about that.

**Comment by djeedai** [comment]
Would you 
And
